chess/events/src/index.ts

```diff
@@ -0,0 +1,84 @@
+import type { EventQueue } from '@paima/events';
+import { generateAppEvents } from '@paima/events';
+import { genEvent } from '@paima/events';
+import { Type } from '@sinclair/typebox';
+
+export const CreatedLobby_v1 = genEvent({
+  name: 'CreatedLobby',
+  fields: [
+    {
+      indexed: true,
+      name: 'user',
+      type: Type.String(),
+    },
+    {
+      indexed: true,
+      name: 'lobbyId',
+      type: Type.String(),
+    },
+    {
+      indexed: false,
+      name: 'rounds',
+      type: Type.Number(),
+    },
+  ],
+} as const);
```

Comment:
as const required?

Reply:
I guess not, I probably added it when testing something.